<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
	border-image:url(:/Wallpaper/wallpaper_3.jpg);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="NavigatorWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#NavigatorWidget
{
	background-color:rgb(44, 52, 63);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="ProgramNameWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#ProgramNameWidget
{
	background-color:rgb(60, 67, 77);
	border-bottom:1px solid rgba(255,255,255,15);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="ProgramName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#ProgramName
{
	font:16px &quot;黑体&quot;;
	color:white;
	padding-left:5px;
}</string>
            </property>
            <property name="text">
             <string>图书馆</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ListWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QRadioButton
{
	height:40px;
	font: 16px &quot;黑体&quot;;
	color:rgb(158, 164, 176);
	padding-left:15px;
}
QLabel
{
	height:30px;
	font:bold 16px &quot;黑体&quot;;
	color:rgb(126, 132, 144);
	padding-left:10px;
}
QRadioButton::indicator
{
	width:17px;
	height:17px;
	margin-right:10px;
}
QRadioButton:checked
{
	color:white;
	background-color:rgba(0,0,0,50);
}
QRadioButton:hover
{
	background-color:rgba(255,255,255,30);
}
QRadioButton:pressed
{
	color:white;
}</string>
         </property>
         <layout class="QVBoxLayout" name="ListLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="HomeButton">
            <property name="styleSheet">
             <string notr="true">#HomeButton
{
	height:60px;
}
#HomeButton::indicator:unchecked
{
	image:url(:/Icon/book1_black.png);
}
#HomeButton::indicator:checked
{
	image:url(:/Icon/book1_white.png);
}</string>
            </property>
            <property name="text">
             <string>主页</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LibraryLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>书库</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BookListButton">
            <property name="styleSheet">
             <string notr="true">#BookListButton::indicator:unchecked
{
	image:url(:/Icon/book2_black.png);
}
#BookListButton::indicator:checked
{
	image:url(:/Icon/book2_white.png);
}</string>
            </property>
            <property name="text">
             <string>图书列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ManagerLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>管理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RecordsButton">
            <property name="styleSheet">
             <string notr="true">#RecordsButton::indicator:unchecked
{
	image:url(:/Icon/book3_black.png);
}
#RecordsButton::indicator:checked
{
	image:url(:/Icon/book3_white.png);
}</string>
            </property>
            <property name="text">
             <string>借阅记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="UsersButton">
            <property name="styleSheet">
             <string notr="true">#UsersButton::indicator:unchecked
{
	image:url(:/Icon/book3_black.png);
}
#UsersButton::indicator:checked
{
	image:url(:/Icon/book3_white.png);
}</string>
            </property>
            <property name="text">
             <string>用户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BooksButton">
            <property name="styleSheet">
             <string notr="true">#BooksButton::indicator:unchecked
{
	image:url(:/Icon/book3_black.png);
}
#BooksButton::indicator:checked
{
	image:url(:/Icon/book3_white.png);
}</string>
            </property>
            <property name="text">
             <string>图书</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="StatisticsLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>统计</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="StatisticsButton">
            <property name="styleSheet">
             <string notr="true">#StatisticsButton::indicator:unchecked
{
	image:url(:/Icon/book3_black.png);
}
#StatisticsButton::indicator:checked
{
	image:url(:/Icon/book3_white.png);
}</string>
            </property>
            <property name="text">
             <string>统计</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <item>
       <widget class="QWidget" name="MainWidget" native="true">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel, QPushButton, QLineEdit
{
	font:14px &quot;黑体&quot;;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="HeadWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>58</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#HeadWidget
{
	background-color:rgba(255,255,255,250);
}
QPushButton
{
	border:2px solid rgb(240, 240, 240);
	background-color:rgb(244, 244, 244);
	border-radius:4px;
}
QPushButton:hover
{
	background-color:rgb(251,251,251);
}
QPushButton:pressed
{
	background-color:rgb(230,230,230);
}</string>
           </property>
           <layout class="QHBoxLayout" name="HeadLayout">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QPushButton" name="TypeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#TypeButton
{
	image:url(:/Icon/more.png);
	padding:3px;
}
#TypeButton::menu-indicator
{
	image:None;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="SearchWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#SearchWidget
{
	border-bottom:2px solid rgb(243, 243, 243);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="TypeLabel">
                 <property name="styleSheet">
                  <string notr="true">#TypeLabel
{
	color:gray;
	padding-right:5px;
}</string>
                 </property>
                 <property name="text">
                  <string>书名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="SearchBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>400</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#SearchBox
{
	border:0;
	font-size:18px;
	background-color: rgba(255,255,255,0);
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="SearchButton">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#SearchButton
{
	border:0;
	image:url(:/Icon/zoom.png);
	padding:5px;
	background-color:rgba(255,255,255,0);
}
#SearchButton:hover
{
	background-color:rgba(0,0,0,30);
}
#SearchButton:pressed
{
	background-color:rgba(0,0,0,60);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="UserName">
              <property name="text">
               <string>// UserName //</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Time">
              <property name="text">
               <string notr="true">// Time //</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UserButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#UserButton
{
	image:url(:/Icon/user.png);
	padding:5px;
}
#UserButton::menu-indicator
{
	image:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="BannerWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65535</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="BannerLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>7</number>
            </property>
            <property name="topMargin">
             <number>7</number>
            </property>
            <property name="rightMargin">
             <number>7</number>
            </property>
            <property name="bottomMargin">
             <number>7</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="TitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">#TitleLabel
{
	font:32px &quot;微软雅黑&quot;;
	color:white;
}</string>
              </property>
              <property name="text">
               <string>你好，%1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="DescribeLabel">
              <property name="styleSheet">
               <string notr="true">#DescribeLabel
{
	color:white;
	font: 16px &quot;微软雅黑&quot;;
}</string>
              </property>
              <property name="text">
               <string>%1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="styleSheet">
            <string notr="true">#stackedWidget
{
	background-color:rgba(255,255,255,230);
}</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page"/>
           <widget class="QWidget" name="page_2"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>HomeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BookListButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UsersButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BooksButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StatisticsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RecordsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangedWidget(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSearchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchBox</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSearchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSearchTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChangedWidget(bool)</slot>
  <slot>onSearchButtonClicked()</slot>
  <slot>onSearchTextChanged(QString)</slot>
 </slots>
</ui>
